crates/cxx-qt-gen/src/generator/cpp/mod.rs

```diff
@@ -11,6 +11,7 @@ pub mod inherit;
 pub mod locking;
 pub mod method;
 pub mod property;
+pub mod qenum;
```

Comment:
Maybe add an entry into the CHANGELOG as well ?

Reply:
I don't want to do that yet, as with Constructor, I think it makes sense to merge this change, but not make it "public" in the CHANGELOG until we've added documentation, support for `namespace`, etc.